core/core.gradle

```diff
@@ -35,17 +35,17 @@ setVersion(project.ext.versionNumber)
 dependencies {
     // localGroovy (which is currently 2.4.x) is used. Groovy 2.5.x supports JsonGenerator which could be used
     // in SipJsonGenerator, but causes issues with downstream builds, so alternatives provided by Groovy 2.4.x are used.
-    compile localGroovy()
-    //compile 'org.codehaus.groovy:groovy-all:2.5.4'
+    //compile localGroovy()
+    compile 'org.codehaus.groovy:groovy-all:2.4.15'
```

Comment:
I find it odd that the groovy dependency doesn't show up in the pom if `compile localGroovy()` is used. But I can't compile on this android table so I can't see what pom actually gets generated. I'll take your word for it. However, please add a comment that localGroovy() is not used because if it is used the groovy dependency will not show up in the pom. Also, leave the original commented out 2.5.4 dependency commented out, as it's important to show that such a dependency exists, but is not used because it messes up gradle builds.